py-polars/polars/series/series.py

```diff
@@ -927,11 +927,9 @@ def __deepcopy__(self, memo: None = None) -> Self:
         return self.clone()
 
     def __contains__(self, item: Any) -> bool:
-        # TODO: optimize via `is_in` and `SORTED` flags
-        try:
-            return (self == item).any()
-        except ValueError:
-            return False
+        if item is None:
+            return self.null_count() > 0
```

Comment:
I'd rather implement this in `.list.contains` directly on the Rust side, but I would need `.list.null_count` which doesn't currently exist.

Reply:
As in a null_count per element?

Reply:
Yes! So:

```python
s = pl.Series([[1,2], [None, 2], [None, None]])
s.list.null_count()  # returns Series([0, 1, 2])
```

I think ideally, we'd have all `Expr` methods also available in the List namespace automatically.